dockerfiles/base/scripts/base/commands/cmd_destroy.sh

```diff
@@ -78,3 +66,12 @@ cmd_destroy() {
   fi
 }
 
+directory_is_empty() {
```

Comment:
It is a bit confusing to have both directory_is_empty and directory_is_clean functions. Is it on purpose?